neuralop/layers/spectral_convolution.py

```diff
@@ -38,33 +45,37 @@ def _contract_dense(x, weight, separable=False):
         # if x is half precision, run a specialized einsum
         return einsum_complexhalf(eq, x, weight)
     else:
-        return tl.einsum(eq, x, weight)
+        return tl.backend.einsum(eq, x, weight)
```

Comment:
@JeanKossaifi I had to add `.backend.` in order to be able to call `einsum()` properly (otherwise `einsum()` was not found in `tensorly`). Is there a better way to get around that problem?

Reply:
Never mind, I found the fix here: https://stackoverflow.com/questions/76368751/how-to-resolve-attributeerror-module-tensorly-has-no-attribute-svd-error